.nycrc

```diff
@@ -1,7 +1,21 @@
 {
   "exclude": [
-    "**/internal/process/write-coverage.js"
+    "coverage/**",
+    "test/**",
+    "tools/**",
+    "benchmark/**",
+    "deps/**"
   ],
-  "compact": false,
-  "reporter": ["html", "text"]
+  "reporter": [
+    "html",
+    "text",
+    "lcov"
+  ],
+  "lines": 95,
```

Comment:
we can ratchet this number up as we improve the linux coverage #s, and it will result in a failure to the action if it drops below the threshold.